api/v3/views/country.rabl

```diff
@@ -46,7 +46,9 @@ if @current_user.access_to?(Country, :country_statistic)
     attributes :pa_land_area, :pa_marine_area,
       :land_area, :percentage_pa_land_cover,
       :percentage_pa_marine_cover, :marine_area,
-      :polygons_count, :points_count
+      :polygons_count, :points_count, :percentage_oecms_pa_marine_cover,
+      :oecms_pa_land_area, :oecms_pa_marine_area, :percentage_oecms_pa_land_cover,
+      :oecm_polygons, :oecm_points, :protected_area_polygons, :protected_area_points
```

Comment:
Since we have a large list of symbols here, can I suggest we list them vertically to make them more scannable, alphabetise them and use %i syntax?

E.g.

```
attributes %i[
 marine_area
 oecm_points
 oecms_pa_marine_area
 pa_land_area
 pa_marine_area
 percentage_oecms_pa_marine_cover
 percentage_oecms_pa_land_cover
 percentage_pa_land_cover
 points_count
 protected_area_polygons
 protected_area_points
]
```

I think this makes it more readable.